.ci/verify-shellcheck

```diff
@@ -0,0 +1,44 @@
+#! /usr/bin/env bash
+
+set -e
+set -o pipefail
+
+FILE_LIST=" \
+    ./install_on_demand/.etcdctl \
+    ./install_on_demand/.table \
+    ./install_on_demand/.shrc \
+    ./install_on_demand/.k9s\
+    ./install_on_demand/.wireguard\
+    ./hacks/print-etcd-cheatsheet\
+    ./hacks/ops-pod\
+    ./hacks/install_etcdctl\
+    ./hacks/install_k9s\
+    .ci/test\
+    .ci/build\
+    ./dotfiles/.bashrc\
+    ./dotfiles/.bash_aliases\
+    ./dotfiles/.bash_profile\
+    ./dotfiles/.bash_prompt\
+    ./.ci/verify-shellcheck\
+    ./.ci/verify-bandit\
+"
+
+if ! command shellcheck 1>&2 >/dev/null; then
```

Comment:
```suggestion
if ! command -v shellcheck 1>&2 >/dev/null; then
```

Without the `-v` flag this will return true even if `shellcheck` exists as it will try to execute the `shellcheck` command which will fail because it does not have the required arguments.

Same for the `bandit` installation.